reactivesocket-client/src/main/java/io/reactivesocket/client/LoadBalancer.java

```diff
@@ -151,22 +250,29 @@ private synchronized void refreshAperture() {
 
         long now = Clock.now();
         boolean underRateLimit = now - lastApertureRefresh > APERTURE_REFRESH_PERIOD;
-        int previous = targetAperture;
         if (avgPending < 1.0 && underRateLimit) {
```

Comment:
yeah, `pendings` is the average (ewma) number of outstanding messages, it is updated all the time.